javalin/src/test/java/io/javalin/testing/JavalinRule.java

```diff
@@ -0,0 +1,41 @@
+package io.javalin.testing;
+
+import io.javalin.Javalin;
+import io.javalin.core.JavalinConfig;
+import org.junit.rules.ExternalResource;
+
+import java.util.function.Consumer;
+
+public class JavalinRule extends ExternalResource {
+    private final Consumer<JavalinConfig> config;
+
+    private Javalin javalin;
+    private HttpUtil http;
+
+    public JavalinRule(Consumer<JavalinConfig> config) {
+        this.config = config;
+    }
+
+    public JavalinRule() {
+        this.config = JavalinConfig.noopConfig;
+    }
```

Comment:
It probably doesn't make much sense to have the non-arg constructor, since test code suppose to setup Javalin before use

Reply:
done